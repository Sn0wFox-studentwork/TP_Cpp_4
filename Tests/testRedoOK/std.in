S s1 0 0 1 1
UNDO
REDO
LIST
EXIT
