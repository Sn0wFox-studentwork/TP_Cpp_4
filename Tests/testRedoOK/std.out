OK
OK
OK
S s1 0 0 1 1
